fusionauth/resource_fusionauth_webhook.go

```diff
@@ -38,6 +38,36 @@ func newWebhook() *schema.Resource {
 				Optional: true,
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
+						"audit_log_create": {
```

Comment:
Sorry for the horrible diffs here, I took the opportunity to order the events the same as the fusion auth docs (alphabetical order) so that it was easy to make sure I hadn't missed any.